interfaces/aux_/aux_.yaml

```diff
@@ -20,6 +20,59 @@ components:
           description: Human-readable message indicating what error occurred and/or why.
           type: string
 
+    DSSInstancesResponse:
+      type: object
+      properties:
+        dss_instances:
+          type: array
+          items:
+            $ref: '#/components/schemas/DSSInstance'
+          default: []
+    DSSInstance:
+      type: object
+      properties:
+        instance_id:
+          description: Identity of this DSS instance participating in the pool (locality).
+          type: string
+        most_recent_heartbeat:
+          description: Most recent heartbeat registered for this DSS instance.
+          anyOf:
+          - $ref: '#/components/schemas/Heartbeat'
+      required:
+      - instance_id
+    Heartbeat:
+      type: object
+      properties:
+        timestamp:
+          description: Time at which heartbeat was registered.
+          type: string
+          format: date-time
+          example: '1985-04-12T23:20:50.52Z'
+        reporter:
+          description: Identity (via access token `sub` claim) of client reporting the heartbeat, or omitted if no client reported the heartbeat.
+          type: string
+          example: 'uss1'
+          default: ''
+        source:
+          description: Source/trigger of this heartbeat.
+          type: string
+          example: Startup
+        index:
```

Comment:
What would that be used for? It does seem to overlap partially with the timestamp.

Reply:
My intent is to use it for resolving update race conditions, with a side benefit of keeping a bit of historical information (for instance, allowing a client to determine the approximate rate at which heartbeats were being made).  In the later PUT, I would expect the heartbeat payload to specify an `index` of {current `index` + 1} and any other values would be rejected 409.

Reply:
Alright, makes sense indeed!